packages/design-system/theme/src/utilities/_float.scss

```diff
@@ -0,0 +1,7 @@
+.float-left {
+  float: left !important;
+}
+
+.float-right {
+  float: right !important;
+}
```

Comment:
don't seem to be used anywhere. might be nice if they are documented in storybook like other utilities function.. (you don't have to put in too much effort there)

Reply:
Added stories for it.